ansible_collections/arista/avd/requirements.txt

```diff
@@ -9,3 +9,5 @@ PyYAML>=5.4.1
 md-toc>=7.1.0
 deepmerge>=1.1.0
 cryptography>=38.0.4
+# Pinned to specific version until anta reaches 1.0.0 and follows semver.
+anta==0.7.2
```

Comment:
will need to to be changed BEFORE we merge this PR to anta 0.8.0 once it is released with all the newer tests added